odl/solvers/util/partial.py

```diff
@@ -135,13 +172,31 @@ class ShowPartial(Partial):
     """Show the partial result."""
 
     def __init__(self, *args, **kwargs):
+        """ Create a show partial
+
+        parameters are passed through to the vectors show method. Additional
+        parameters include
+
+        Parameters
+        ----------
+        *args, **kwargs
+            passed ax ``x.show(*args, **kwargs)``
+        display_step : positive int
+            Plot every n:th image, default: 1
```

Comment:
Swedish ordinal :-) Perhaps write "Number of iterations between plots. Default: 1" (default always as a new "sentence")